client/src/app/scenes/home/index.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import { PageLayout } from '../../components';
+import JackpotInfo from './JackpotInfo';
+import Plays from './Plays';
+import Modal from '../play-modal/modal';
+import { useState } from 'react';
+
+const StyledPageLayout = styled(PageLayout)(() => ({
+	marginTop: 60,
+}));
+
+const HomeScene = () => {
+	const [showModal, setShowModal] = useState<boolean>(false);
+	return (
+		<>
+			<JackpotInfo setModalOpen={setShowModal} />
+			<StyledPageLayout title={'Casper Lottery'}>
+				<Plays setModalOpen={setShowModal} />
+				<Modal isOpen={showModal} setModalOpen={setShowModal} />
+			</StyledPageLayout>
```

Comment:
Here is a reason to use `useCallback` :) 
If we need to pass callback from state, need to wrap it in `useCallback`

But, I'm not sure if we need to pass `setShowModal` inside each of this component and manage state there. Probably better to do smth like that:

```
// here we can pass optional param if necessary
const handleGameStart = ( parameter ) => {
 setShowModal(true);
}
...
<JackpotInfo onGameStart={handleGameStart} />
```